screens/ShopSearch.js

```diff
@@ -1,110 +1,127 @@
 import React from 'react';
-import { StyleSheet, Text, View, Image, Dimensions } from 'react-native';
+import { StyleSheet, Text, View, Image, Dimensions, TextInput, TouchableOpacity} from 'react-native';
 import OrangeBackground from './../components/OrangeBackground';
+import FontLoad from './../components/FontLoad';
+import Toolbar from './../components/Toolbar';
 
+const window = Dimensions.get("window")
 
 export default class ShopSearch extends React.Component {
   static navigationOptions = {
     header: null,
     };
     
+    search = () => {
+      this.props.navigation.navigate('Results')
+    }
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        fontLoaded: false,
+      }
+    }
+  
+    componentWillMount = () => {
+      FontLoad.then((res) => {
+        this.setState({ fontLoaded: true });
+      })
+    }
+    
   render() {
     return (
-      <View style={styles.container}>
-        <OrangeBackground/>
-        <Text style={styles.header}>Let{`'`}s shopU! </Text>
-        <View style={styles.slantedBlueRectangle}></View>
-        <View style={styles.slantedPurpleRectangle}></View>
-        <View style={styles.whiteCard}></View>
-        <View style={styles.searchLine}></View>
-        <Text style={styles.searchText}>Search for a Product</Text>
-        <Image source={require('./../assets/images/search_icon.png')}style={styles.searchIcon} />
-        <Image source={require('./../assets/images/top_shopping_cart.png')}style={styles.topShopCart} />
-        <Image source={require('./../assets/images/profile_head.png')}style={styles.profileHead} />
-      </View>
+      this.state.fontLoaded ? (
+        <View style={styles.container}>
+          <OrangeBackground/>
+          <View style={{marginTop: window.height*.015}}>
```

Comment:
Why did you add this styling? You shouldn't have to add extra styling around the `Toolbar` component